pkg/container-collection/k8s.go

```diff
@@ -181,3 +192,42 @@ func (k *K8sClient) ListContainers() (arr []Container, err error) {
 	}
 	return arr, nil
 }
+
+func getCurrentKubeletConfig(clientset *kubernetes.Clientset, nodeName string) (*kubeletconfig.KubeletConfiguration, error) {
```

Comment:
what are the required permissions for requesting the kubelet config?

Reply:
Good call. We need to add `get` for `nodes/proxy` in the cluster role

Reply:
* Update charts/gadget/templates/clusterrole.yaml
* `make generate-manifests`